blivet/formats/disklabel.py

```diff
@@ -228,7 +228,7 @@ def get_platform_label_types(cls):
         elif arch.is_efi() and arch.is_arm():
             label_types = ["msdos", "gpt"]
         elif arch.is_efi() and not arch.is_aarch64():
-            label_types = ["gpt"]
+            label_types = ["gpt", "msdos"]
```

Comment:
The two `elif` branches for `is_efi()` could be merged into a single one now.

Reply:
The first one in the list is the default, in teh case of is_arm we need msdos as the default but x86 we obviously want gpt as the default. I had left the checks as they were due to arch defaults (this has been broken by others in the past).

Reply:
Makes sense, thanks for the explanation!